pre-docs/bolt_options.md

```diff
@@ -193,6 +194,7 @@ Displays a list of all the tasks on the modulepath. Will note whether a task sup
 bolt plan show
 ```
 - Lists the plans that are installed on the current module apth
```

Comment:
Typo: `apth` should be `path`. Also, could use a period at the end to be consistent.